app/controllers/decidim/proposals/proposals_controller.rb

```diff
@@ -35,6 +35,14 @@ def index
 
         @proposals = paginate(@proposals)
         @proposals = reorder(@proposals)
+
+        # TODO esto es nuevo
```

Comment:
Remove usless comments and commented code

Reply:
Removed